### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mkostevich**

### CODE REVIEW

File: Model/Resolver/Cart/AbstractAppliedFees.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        $this->priceCurrency = $priceCurrency;
+        $this->enumLookup    = $enumLookup;
+    }
+
+    /**
+     * @param array $options
+     * @param string $currencyCode
+     * @return array
+     */
+    protected function getPreparedOprions(array $options, string $currencyCode): array
```
Comment (SiarheyUchukhlebau) [inline Model/Resolver/Cart/AbstractAppliedFees.php]: Typo: `getPreparedOprions` must be `getPreparedOptions`

File: etc/schema.graphqls [hunk 2]
```diff
@@ -0,0 +1,103 @@
+# Copyright © MageWorx, Inc. All rights reserved.
+# See https://www.mageworx.com/terms-and-conditions for license details.
+
+type Cart {
+    mw_applied_quote_fees: [MwAppliedFee] @doc(description: "Applied Quote Fees") @resolver (class: "\\MageWorx\\MultiFeesGraphQl\\Model\\Resolver\\Cart\\AppliedQuoteFees")
+    mw_applied_product_fees: [MwAppliedProductFee] @doc(description: "Applied Product Fees") @resolver (class: "\\MageWorx\\MultiFeesGraphQl\\Model\\Resolver\\Cart\\AppliedProductFees")
```
Comment (SiarheyUchukhlebau) [inline etc/schema.graphqls:17]: Move `MwAppliedProductFee` from the `Cart` level to the `CartItem` level.

**Review by SiarheyUchukhlebau — CHANGES_REQUESTED** [review]

File: Model/Resolver/Cart/AbstractAppliedFees.php (hunk 1 above)
Comment (mkostevich) [reply Model/Resolver/Cart/AbstractAppliedFees.php]: done

File: etc/schema.graphqls (hunk 2 above)
Comment (mkostevich) [reply etc/schema.graphqls:17]: done.

**Review by SiarheyUchukhlebau — APPROVED** [review]

**Pull request merged**